### ISSUE TITLE
Start WildFly in Full Platform instead of Web Profile

### ISSUE THREAD

**Issue by arun-gupta** [CentOS/CentOS-Dockerfiles#19]
WildFly, by default, starts in Web Profile. This means the technologies that are from the Java EE 7 Full Platform, such as JMS and Batch, cannot be used.

A sample to start WildFly in Full Platform mode is shown at:

https://github.com/arun-gupta/docker-images/blob/master/wildfly-centos/Dockerfile

**Comment by jperrin** [comment]
I've not intentionally ignored this, but I apologize for the delay in addressing it. Is the dockerfile you linked to still how you would like to see it set up? I'm going to be merging/updating for the next few days.

**Comment by arun-gupta** [comment]
Yes, https://github.com/arun-gupta/docker-images/blob/master/wildfly-centos/Dockerfile#L47 shows exactly the command to start WildFly.

**Comment by jperrin** [comment]
In theory this should be updated now. I've changed the repository around so that it's an automated build in the docker index. Once it's built, the change should be in place. If there's anything else you'd like to see, please submit a pull request.
